opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectsTimeline.jsx

```diff
@@ -43,22 +43,9 @@ const stixCoreObjectsTimelineQuery = graphql`
             x_opencti_order
             x_opencti_color
           }
-          ... on BasicObject {
-            id
-            entity_type
-          }
-          ... on StixObject {
-            representative {
-              main
-              secondary
-            }
-          }
-          ... on StixDomainObject {
-            created
-            modified
-          }
-          ... on StixCyberObservable {
-            observable_value
```

Comment:
`observable_value` is also used in the function to get representatives

Reply:
we don't need it if we get representatives directly from the API

Reply:
Or we do not need anymore as we retrieve representatives directly from backend ?